src/app/tests/suites/certification/Test_TC_G_2_3.yaml

```diff
@@ -589,7 +589,7 @@ tests:
       verification: |
           ./chip-tool groupkeymanagement read group-table 1 0
 
-          Verify the "grouptable" does not have GroupName  Gp9 on the TH(Chip-tool)  Log and below is the sample log provided for the raspi platform:
+          Verify the "grouptable" does not have GroupName  gp9 on the TH(Chip-tool)  Log and below is the sample log provided for the raspi platform:
```

Comment:
It should be Verify the "grouptable" does not have GroupName gp5 on the TH(Chip-tool) Log and below is the sample log provided for the raspi platform: